frontend/src/metabase/components/EntityIdCard/EntityIdCard.tsx

```diff
@@ -50,7 +50,14 @@ export function EntityIdCard({ entityId }: { entityId: string }) {
     <SidesheetCard title={<EntityIdTitle />}>
       <Flex gap="sm" align="end">
         <Text>{entityId}</Text>
-        <CopyButton value={entityId} />
+        <Box
+          component={CopyButton}
+          value={entityId}
+          c="var(--mb-color-text-light)"
+          style={{ cursor: "pointer" }}
+          pos="relative"
+          top={-2}
+        />
```

Comment:
this copy button should have a hover style that sets the color to brand, which cannot be achieved with style props alone